README.md

```diff
@@ -23,23 +23,38 @@
 
 `yarn add @palmetto/palmetto-components`
 
-### 2. import global css
+### 2. Import Global CSS
 
 ```
 @import '@palmetto/palmetto-components/dist/css/utilities.css';
 @import '@palmetto/palmetto-components/dist/css/variables.css';
 ```
 
-### 3. Use in React component
+### 3. Usage
 
-`import { TextInput } from '@palmetto/palmetto-components'`
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '@palmetto/palmetto-components';
```

Comment:
Hey @JCarmine importing without destructuring won't work as there is no default import in the library, only individual named imports for each component.